python/bindings/openravepy_int.cpp

```diff
@@ -3791,6 +3801,172 @@ Because race conditions can pop up when trying to lock the openrave environment
         env.attr("TriangulateOptions") = selectionoptions;
     }
 
+#ifdef USE_PYBIND11_PYTHON_BINDINGS
+    object lengthUnit = enum_<LengthUnit>(m, "LengthUnit" DOXY_ENUM(LengthUnit))
+#else
+    object lengthUnit = enum_<LengthUnit>("LengthUnit" DOXY_ENUM(LengthUnit))
+#endif
+                          .value("Meter", LU_Meter)
```

Comment:
please use GetLengthUnitString() for the strings rather than hardcoding them like this

Reply:
If so, the unit enum name will be `openravepy.LengthUnit.m` when we use it in python. I feel like `openravepy.LengthUnit.Meter` is more readable.